### DISCUSSION TITLE
not able to build it on your own

### DISCUSSION THREAD

**Discussion by vabov** [cafali/SnapKey#37] [General]
![image](https://github.com/user-attachments/assets/d3b48bfd-00a5-4e96-8130-1998ce188ff5)
yeah...

**Comment by cafali** [comment]
Since cmake is not recognized as a command, ensure that the directory containing cmake.exe is added to your system’s PATH environment variable. This will allow your system to locate and execute CMake from any command prompt or terminal.

**Comment by vabov** [comment]
interesting alright, also there's another problem. i have tested this software on postal 2 and it worked magnificently but when i tried it on counter strike source for some reason it did not work. any reason for this? maybe im doing something wrong?

**Reply by vabov** [reply]
and uhh roblox.. (yeah ik dissapointing)
